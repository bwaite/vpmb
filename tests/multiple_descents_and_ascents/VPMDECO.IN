TRIMIX DIVE TO 80 MSW
1
0.15,0.45,0.4
1
0,80,23,1,0.7
2
80,30,1,1.3
1
80,60,-23,1,0.7
2
60,45,1,1.3
1
60,90,23,1,0.7
2
90,60,1,1.3
99
1
90,1,-10,3,1.6
0
