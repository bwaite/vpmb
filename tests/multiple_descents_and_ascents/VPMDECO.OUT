                          DECOMPRESSION CALCULATION PROGRAM
                        Developed in FORTRAN by Erik C. Baker
 
 
Program Run:    03-27-2010 at 10:47 am                       Model: VPM-B
 
Description:    TRIMIX DIVE TO 80 MSW                                                 
 
Gasmix Summary:                        FO2    FHe    FN2
                          Gasmix # 1  0.150  0.450  0.400
 
 
                                    DIVE PROFILE
 
Seg-  Segm.  Run   | Gasmix | Ascent    From     To      Rate    | Constant
ment  Time   Time  |  Used  |   or     Depth   Depth    +Dn/-Up  |  Depth
  #   (min)  (min) |    #   | Descent  (mswg)  (mswg)  (msw/min) |  (mswg)
----- -----  ----- | ------ | -------  ------  ------  --------- | --------
  1     3.5    3.5 |    1   | Descent     0.     80.      23.0   |
  2    26.5   30.0 |    1   |                                    |    80.
  3     0.9   30.9 |    1   | Ascent     80.     60.     -23.0   |
  4    14.1   45.0 |    1   |                                    |    60.
  5     1.3   46.3 |    1   | Descent    60.     90.      23.0   |
  6    13.7   60.0 |    1   |                                    |    90.
 
 
                               DECOMPRESSION PROFILE
 
          Leading compartment enters the decompression zone at   71.6 mswg
                 Deepest possible decompression stop is   69.0 mswg
 
Seg-  Segm.  Run   | Gasmix | Ascent   Ascent   Col   |  DECO   STOP   RUN
ment  Time   Time  |  Used  |   To      Rate    Not   |  STOP   TIME   TIME
  #   (min)  (min) |    #   | (mswg) (msw/min)  Used  | (mswg)  (min)  (min)
----- -----  ----- | ------ | ------ --------- ------ | ------  -----  -----
  7     2.7   62.7 |    1   |   63.    -10.0          |
  8     0.3   63.0 |    1   |                         |    63      1     63
  9     0.3   63.3 |    1   |   60.    -10.0          |
 10     0.7   64.0 |    1   |                         |    60      1     64
 11     0.3   64.3 |    1   |   57.    -10.0          |
 12     0.7   65.0 |    1   |                         |    57      1     65
 13     0.3   65.3 |    1   |   54.    -10.0          |
 14     1.7   67.0 |    1   |                         |    54      2     67
 15     0.3   67.3 |    1   |   51.    -10.0          |
 16     1.7   69.0 |    1   |                         |    51      2     69
 17     0.3   69.3 |    1   |   48.    -10.0          |
 18     2.7   72.0 |    1   |                         |    48      3     72
 19     0.3   72.3 |    1   |   45.    -10.0          |
 20     2.7   75.0 |    1   |                         |    45      3     75
 21     0.3   75.3 |    1   |   42.    -10.0          |
 22     3.7   79.0 |    1   |                         |    42      4     79
 23     0.3   79.3 |    1   |   39.    -10.0          |
 24     3.7   83.0 |    1   |                         |    39      4     83
 25     0.3   83.3 |    1   |   36.    -10.0          |
 26     5.7   89.0 |    1   |                         |    36      6     89
 27     0.3   89.3 |    1   |   33.    -10.0          |
 28     6.7   96.0 |    1   |                         |    33      7     96
 29     0.3   96.3 |    1   |   30.    -10.0          |
 30     7.7  104.0 |    1   |                         |    30      8    104
 31     0.3  104.3 |    1   |   27.    -10.0          |
 32    10.7  115.0 |    1   |                         |    27     11    115
 33     0.3  115.3 |    1   |   24.    -10.0          |
 34    12.7  128.0 |    1   |                         |    24     13    128
 35     0.3  128.3 |    1   |   21.    -10.0          |
 36    17.7  146.0 |    1   |                         |    21     18    146
 37     0.3  146.3 |    1   |   18.    -10.0          |
 38    22.7  169.0 |    1   |                         |    18     23    169
 39     0.3  169.3 |    1   |   15.    -10.0          |
 40    29.7  199.0 |    1   |                         |    15     30    199
 41     0.3  199.3 |    1   |   12.    -10.0          |
 42    44.7  244.0 |    1   |                         |    12     45    244
 43     0.3  244.3 |    1   |    9.    -10.0          |
 44    68.7  313.0 |    1   |                         |     9     69    313
 45     0.3  313.3 |    1   |    6.    -10.0          |
 46   110.7  424.0 |    1   |                         |     6    111    424
 47     0.3  424.3 |    1   |    3.    -10.0          |
 48   203.7  628.0 |    1   |                         |     3    204    628
 49     0.3  628.3 |    1   |    0.    -10.0          |
 
